quail/processes/wps_climdex_days.py

```diff
@@ -12,7 +12,15 @@
 class ClimdexDays(Process):
     """
     Takes a climdexInput object as input and computes the annual count
-    of days where daily maximum temperature satisfies some condition
+    of days where daily temperature satisfies some condition.
+    - "summer": the annual count of days where daily maximum temperature
+    exceeds 25 degreesCelsius
+    - "icing": the annual count of days where daily maximum temperature
+    was below 0 degrees Celsius
+    - "frost": the annual count of days where daily minimum temperature
+    was below 0 degrees Celsius
+    - "tropical nights": the annual count of days where daily minimum
+    temperature stays above 20 degrees Celsius
```

Comment:
It may be useful to the user to include a description like this in the abstract.